packages/app/src/sections/dashboard/FuturesMarketsTable.tsx

```diff
@@ -85,26 +99,27 @@ const FuturesMarketsTable: React.FC<FuturesMarketsTableProps> = ({ search }) =>
 		})
 	}, [search, futuresMarkets, t, futuresVolumes, pricesInfo, pastRates, markPrices])
 
+	const sortBy = useMemo(() => [{ id: 'dailyVolume', desc: true }], [])
```

Comment:
We could just move this outside the scope of the component rather than use memo

Reply:
True. Moved.